app/src/main/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivity.kt

```diff
@@ -16,19 +16,19 @@ class CompletedStoryListActivity : InjectableAppCompatActivity() {
     super.onCreate(savedInstanceState)
     (activityComponent as ActivityComponentImpl).inject(this)
     val internalProfileId: Int =
-      intent.getIntExtra(COMPLETED_STORY_LIST_ACTIVITY_PROFILE_ID_KEY, -1)
+      intent.getIntExtra(COMPLETED_STORY_LIST_ACTIVITY_PROFILE_ID_EXTRA_KEY, -1)
```

Comment:
As per mentioned in the issue description:

```
For activities:
<PROPERTY_NAME>_EXTRA_KEY = "<FileName>.<property_name>"
Example: INTERNAL_PROFILE_ID_EXTRA_KEY = "SampleActivity.internal_profile_id"
```

we are including only `<PROPERTY_NAME>_EXTRA_KEY` not the activity name - `COMPLETED_STORY_LIST_ACTIVITY`!!

Deferring to @rt4914 for clarification.

Reply:
Make sense. It should be 

`const val PROFILE_ID_EXTRA_KEY = "CompletedStoryListActivity.profile_id"

Reply:
PTAL here #3880